docs/migrating-to-velero.md

```diff
@@ -0,0 +1,47 @@
+# Migrating from Ark to Velero
+
+Users of Ark will need to move their installation to the renamed Velero images and Custom Resource Definitions (CRDs). This is a one-time operation.
+
+At a high level, this process involves the following steps:
+
+    * Scale down the `ark` deployment, so it will not process schedules, backups, or restores during the migration period.
+    * Apply the new CRDs.
```

Comment:
maybe explicitly add a bullet stating that a new namespace `velero` will be created and all objs will move there

Reply:
Added - we may want to think about this for users who have changed the default namespace.

I think providing instructions with documented defaults for now should be ok, what do you think?